File: lib/src/model/image_persist_request.dart
import 'oss_request.dart';

class ImagePersistRequest extends OSSRequest {
  String fromBucket;

  String fromObjectKey;

  String toBucketName;

  String toObjectKey;

  String action;

  ImagePersistRequest(this.fromBucket, this.fromObjectKey, this.toBucketName,
      this.toObjectKey, this.action);
}

File: lib/src/model/ImagePersistRequest.java
package com.alibaba.sdk.android.oss.model;

/**
 * Created by huaixu on 2018/1/29.
 */

 class ImagePersistRequest extends OSSRequest {

     String mFromBucket;

     String mFromObjectkey;

     String mToBucketName;

     String mToObjectKey;

     String mAction;


     ImagePersistRequest(String fromBucket, String fromObjectKey, String toBucketName, String mToObjectKey, String action) {
        this.mFromBucket = fromBucket;
        this.mFromObjectkey = fromObjectKey;
        this.mToBucketName = toBucketName;
        this.mToObjectKey = mToObjectKey;
        this.mAction = action;
    }

}
